### DISCUSSION TITLE
Debian 10 link errors.

### DISCUSSION THREAD

**Discussion by rfoxkendo** [fltk-rs/fltk-rs#1292] [General]
More a note than anything.  Deb10 packages for some of the dependencies appear to be broken and this results in link errors.  The specific breakage is the lack of symblolic links for libxxxx.so pointing to several actual libraries  this can be worked around by a root person as follows:
```
LIB=/usr/lib/x86_64-linux-gnu

ln -s $LIB/libXinerama.so.1     $LIB/libXinerama.so
ln -s $LIB/libXcursor.so.1      $LIB/libXcursor.so
ln -s $LIB/libpango-1.0.so.0    $LIB/libpango-1.0.so
ln -s $LIB/libpangoxft-1.0.so.0 $LIB/libpangoxft-1.0.so
ln -s $LIB/libgobject-2.0.so.0  $LIB/libgobject-2.0.so
ln -s $LIB/libcairo.so.2        $LIB/libcairo.so
ln -s $LIB/libpangocairo-1.0.so.0 $LIB/libpangocairo-1.0.so
```

I have to pull together some time to make the bug report(s) as this problem spans several packages...for non-intel debian installations you'll need to change the definition of LIB above appropriately.  Deb-11 fixes this problem.

**Comment by MoAlyousef** [comment]
Thank you for the heads up

**Comment by rfoxkendo** [comment]
Did a bit more poking around.  This is fixed by installing the package:libxfixes-dev As well. RF Sent from Mail for Windows From: Mohammed AlyousefSent: Wednesday, October 5, 2022 9:53 AMTo: fltk-rs/fltk-rsCc: Ron Fox; AuthorSubject: Re: [fltk-rs/fltk-rs] Debian 10 link errors. (Discussion #1292) Thank you for the heads up—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
